sphinx_gallery/gen_rst.py

```diff
@@ -234,6 +235,12 @@ def __exit__(self, type_, value, tb):
       :download:`Download Jupyter notebook: {0} <{0}>`
 """
 
+ZIP_DOWNLOAD = """
+    .. container:: sphx-glr-download sphx-glr-download-zip
+
+      :download:`Download all: {0} <{0}>`
```

Comment:
Somewhere else I think we have a "download all" which means "download from all examples", so I think it's less ambiguous to say "download both" here or "download zipped ipynb and py files" or something. So a minimal version would be 
```suggestion
      :download:`Download both (zipped): {0} <{0}>`
```